examples_extra/ASDKgram-Swift/ASDKgram-Swift.xcodeproj/project.pbxproj

```diff
@@ -428,7 +428,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```

Comment:
The changes in this file are unintentional I guess?

Reply:
No this was intentional. It doesn't compile otherwise.

Reply:
👍